service.go

```diff
@@ -101,11 +101,11 @@ type ZomeFile struct {
 	Name         string
 	Description  string
 	CodeFile     string
-	Entries      []EntryDefFile
 	RibosomeType string
-	Functions    []FunctionDef
 	BridgeFuncs  []string // functions in zome that can be bridged to by fromApp
 	BridgeTo     string   // dna Hash of toApp that this zome is a client of
+	Entries      []EntryDefFile
+	Functions    []FunctionDef
```

Comment:
The reason I reorganized these is because it's nicer to have all the short properties at the top, not scattered. True for the DNA at the top level, and also the Zomes